client/client-multi/client-c/src/kaa/kaa_logging.h

```diff
@@ -42,6 +42,54 @@ extern "C" {
     typedef struct kaa_log_collector        kaa_log_collector_t;
 #endif
 
+/**
```

Comment:
Please, move kaa_log_bucket_info_t and kaa_log_listeners_t () to a separate header in the "platform" directory (something like ext_log_delivery_listener.h) to follow the approach that is already used in configuration, notification and event features.